BioCro/DARPA/biocro_regional_darpa.md

```diff
@@ -0,0 +1,132 @@
+How to Do Regional Runs on DARPA Data Using BioCro Model
+================
+Kristina Riemer, University of Arizona
```

Comment:
Your name under the title looks kind of funny (as in strange). You might want to prefix it with `**Author**:` or something like that